spec/unit/patterns_inspector_spec.rb

```diff
@@ -125,6 +125,13 @@
         expect { patterns_inspector.inspect(filter) }.to raise_error(
           Machinery::Errors::ZypperFailed, /Zypper is locked./)
       end
+
+      it "returns patterns_system zypper" do
+        expect(system).to receive(:run_command).and_return(zypper_output)
+        patterns_inspector.inspect(filter)
+        patterns_system = description.patterns.attributes["patterns_system"]
```

Comment:
I don't think two lines are needed here unless it is bigger than 100 characters.